src/utils/venue.ts

```diff
@@ -28,6 +28,9 @@ export const canBeDeleted = (venue: AnyVenue): boolean =>
 export const canHavePlacement = (venue: AnyVenue): boolean =>
   PLAYA_TEMPLATES.includes(venue.template);
 
+export const checkIfValidVenueId = (venueId: string): boolean =>
+  /[a-z0-9_]{1,250}/.test(venueId);
+
```

Comment:
Can we also move the function that generates `venueId`s from the `venueName` out of the specific admin component/etc file it currently lives in, and have it here alongside this check. That way we'll increase the likelihood that they remain properly in sync if/when changes are made to one/the other.

Bonus points if you also then import that function onto the `scripts/` script that uses the same code, so that it uses the util from here too.

Bonus bonus points if you add `@debt` comments to the definitions of both functions here + their equivalent in `functions/venue` referencing the other, and saying how we should unify them when possible (there should be existing debt comments like this in `functions/venue`, so you should be able to copy that same style)

Reply:
OOS

Reply:
This is one of those cases where I'm going to get frustrated at you calling 'OOS' when it will take like 2 seconds to do it. There are 2 functions, used in barely any places, and a comment to add.

It would take you longer to add a `@debt` comment calling out that this should be done in future, or to backlog it as a future task.

Reply:
Nothing takes 2 seconds. Nothing ever takes less than an hour realistically.

Reply:
@jonboutelle Thanks for that insight... But I spent longer typing out https://github.com/sparkletown/sparkle/pull/1299#discussion_r626252430 than it would take to fix this.. so.. thanks for that... super helpful..